tests/test_vis.py

```diff
@@ -497,3 +497,18 @@ def test_matplotlib_heatmap_flag_config():
     assert not df.recommendation["Correlation"][0]._postbin
     lux.config.heatmap = True
     lux.config.plotting_backend = "vegalite"
+
+
+def test_unnamed_column():
+    lux.config.plotting_backend = "matplotlib"
+    df = pd.read_csv("lux/data/car.csv")
+    df.plot_config = None
+    df = df.rename(columns={"Horsepower": " "})
```

Comment:
Can we change this line to the original example to make it more realistic? 
```python
df = pd.read_csv("https://raw.githubusercontent.com/lux-org/lux-datasets/master/data/employee.csv")
df["YearsAtCompany"]/df["TotalWorkingYears"]
```

Reply:
im not exactly sure how to get the vis.code for a LuxSeries object. is there a way to do this?

Reply:
fixed my adding recommendation attribute to luxSeries https://github.com/lux-org/lux/pull/266